framework/src/Volo.Abp.MongoDB/Volo/Abp/Domain/Repositories/MongoDB/IMongoDbBulkOperationProvider.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Volo.Abp.Domain.Entities;
+using Volo.Abp.Domain.Repositories.MongoDB;
+
+namespace Volo.Abp.MongoDB.Volo.Abp.Domain.Repositories.MongoDB
+{
+    public interface IMongoDbBulkOperationProvider
+    {
+        Task InsertManyAsync<TEntity>(
+           IMongoDbRepository<TEntity> repository,
+           IEnumerable<TEntity> entities,
+           bool autoSave,
+           CancellationToken cancellationToken
+       )
+           where TEntity : class, IEntity;
+
+
+        Task UpdateManyAsync<TEntity>(
+            IMongoDbRepository<TEntity> repository,
+            IEnumerable<TEntity> entities,
+            bool autoSave,
+            CancellationToken cancellationToken
+        )
+            where TEntity : class, IEntity;
+
+
```

Comment:
Use single empty line, not two empty lines between methods

Reply:
Can you check Commit f3722985